tests/ga/test_extension.py

```diff
@@ -3335,26 +3435,25 @@ def manifest_location_handler(url, **kwargs):
 
                     if wrapped_url and location_uri_regex.match(wrapped_url):
                         return Exception("Ignoring host plugin requests for testing purposes.")
-                
+
                 return None
-            
+
             if manifest_location_handler.num_times_called == 0:
                 time.sleep(.3)
                 manifest_location_handler.num_times_called += 1
-                return Exception("Failing manifest fetch from uri '{0}' for testing purposes."\
-                    .format(url))
-        
+                return Exception("Failing manifest fetch from uri '{0}' for testing purposes." \
+                                 .format(url))
+
             return None
-        
-        manifest_location_handler.num_times_called = 0
 
+        manifest_location_handler.num_times_called = 0
 
         with mock_wire_protocol(self.test_data, http_get_handler=manifest_location_handler) as protocol:
             ext_handlers, _ = protocol.get_ext_handlers()
 
             with self.assertRaises(ExtensionDownloadError):
                 protocol.client.fetch_manifest(ext_handlers.extHandlers[0].versionUris,
-                    timeout_in_minutes=0, timeout_in_ms=200)
+                                               timeout_in_minutes=0, timeout_in_ms=200)
```

Comment:
Thanks for doing these cosmetic changes! I hope they're at-par with the pep-8 standards!

Reply:
This created issues with pylint checks , have to undo most of it unfortunately :)